cypress/integration/options/axes.cy.js

```diff
@@ -86,19 +88,60 @@ describe('Options - Vertical axis', () => {
         })
     })
     // TODO: steps, decimals, labels
-    describe('options modal keeps changes', () => {
+    describe('options modal keeps changes when reopening', () => {
         it('opens Options -> Axes', () => {
             clickMenuBarOptionsButton()
             clickOptionsTab(OPTIONS_TAB_AXES)
         })
         it(`title is "${TEST_TITLE}"`, () => {
-            expectVerticalAxisTitleToBeValue(TEST_TITLE)
+            expectAxisTitleToBeValue(TEST_AXIS, TEST_TITLE)
         })
         it(`range min is "${TEST_MIN_VALUE}"`, () => {
-            expectVerticalAxisRangeMinToBeValue(TEST_MIN_VALUE)
+            expectAxisRangeMinToBeValue(TEST_AXIS, TEST_MIN_VALUE)
         })
         it(`range max is "${TEST_MAX_VALUE}"`, () => {
-            expectVerticalAxisRangeMaxToBeValue(TEST_MAX_VALUE)
+            expectAxisRangeMaxToBeValue(TEST_AXIS, TEST_MAX_VALUE)
+        })
+    })
+})
+
+describe('Options - Horizontal axis', () => {
+    const TEST_AXIS = 'DOMAIN_0'
+    const TEST_TITLE = 'Hori title'
+    it('navigates to the start page and add a data item', () => {
+        goToStartPage()
+        openDimension(DIMENSION_ID_DATA)
+        selectDataElements([TEST_DATA_ELEMENT_NAME])
+        clickDimensionModalUpdateButton()
+        expectVisualizationToBeVisible(VIS_TYPE_COLUMN)
+    })
+    describe('title', () => {
+        it('opens Options -> Axes', () => {
+            clickMenuBarOptionsButton()
+            clickOptionsTab(OPTIONS_TAB_AXES)
+        })
+        it('enable title', () => {
+            enableAxisTitle(TEST_AXIS)
+        })
+        it('set custom title', () => {
+            setAxisTitle(TEST_AXIS, TEST_TITLE)
+        })
+        it('click the modal update button', () => {
+            clickOptionsModalUpdateButton()
+            expectChartTitleToBeVisible()
+        })
+        it(`config has vertical axis title "${TEST_TITLE}"`, () => {
```

Comment:
Just confirm that you are checking the `vertical` axis with the function `expectWindowConfigXAxis`

Reply:
Good find, copy/paste error, the text should say `config has horizontal...`